src/rgb_packet_processor.cpp

```diff
@@ -50,21 +53,31 @@ void RgbPacketProcessor::setFrameListener(libfreenect2::FrameListener *listener)
 }
 
 DumpRgbPacketProcessor::DumpRgbPacketProcessor()
+  : frame(NULL)
 {
 }
 
 DumpRgbPacketProcessor::~DumpRgbPacketProcessor()
 {
+  delete frame;
 }
 
 void DumpRgbPacketProcessor::process(const RgbPacket &packet)
 {
-  //std::stringstream name;
-  //name << packet->sequence << "_" << packet->unknown0 << "_" << jpeg_buffer_length << ".jpeg";
-  //
-  //std::ofstream file(name.str().c_str());
-  //file.write(reinterpret_cast<char *>(packet->jpeg_buffer), jpeg_buffer_length);
-  //file.close();
+  frame = new Frame(1, 1, 1920*1080*4);
```

Comment:
Still need this:
```
  frame->bytes_per_pixel = packet.jpeg_buffer_length;
```
`1920*1080*4` is the size for the allocation, but not for the actual data.